### ISSUE TITLE
Could not find the module

### ISSUE THREAD

**Issue by landsman** [KajSzy/next-cache-toolbar#4]
Next 14.1.0

```node
 Internal error: Error: Could not find the module "/Users/landsman/projects/brand/product/frontend/app/node_modules/next-cache-toolbar/dist/es/cache-panel-trigger-client-GrlxjSO5.js#C" in the React Client Manifest. This is probably a bug in the React Server Components bundler.
    at /Users/landsman/projects/brand/product/frontend/app/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:35:265783
    at eM (/Users/landsman/projects/brand/product/frontend/app/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:35:265978)
    at /Users/landsman/projects/brand/product/frontend/app/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:35:277371
    at Array.toJSON (/Users/landsman/projects/brand/product/frontend/app/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:35:281723)
    at stringify (<anonymous>)
    at /Users/landsman/projects/brand/product/frontend/app/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:35:268079
    at ez (/Users/landsman/projects/brand/product/frontend/app/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:35:268158)
    at eH (/Users/landsman/projects/brand/product/frontend/app/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:35:268559)
    at Timeout._onTimeout (/Users/landsman/projects/brand/product/frontend/app/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:35:265057)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)

```
```

**Comment by KajSzy** [comment]
I've made some changes with bundling but have some blocker with `bunchee` concating server only and client only files into one.
As soon as this will get fixed, I will publish new version that should fix this issue

**Comment by KajSzy** [comment]
Should be fixed in `0.2.1`
